debian/changelog

```diff
@@ -1,3 +1,10 @@
+regolith-wm-config (4.1.0-3ubuntu1) UNRELEASED; urgency=medium
+
+  * fix(regolith-desktop/#866): incorrect path for i3 user config directory
+  * feat: add common directory for shared configs between i3 and sway
+
+ -- Soumya Ranjan Patnaik <soumyaranjan1812@gmail.com>  Mon, 26 Jun 2023 18:54:54 +0530
```

Comment:
I confess to not being crisp on the details, but the package signing stuff w/ the regolith key seems to only work if the regolith email address is specified here.  I'll update post merge.  (FYI)  Generally I prefer having this in the PR rather than me adding it later, as the narrative is the entry into the changelog.